modules/rosa-create-objects.adoc

```diff
@@ -0,0 +1,441 @@
+
+|--private
+|Restricts master API endpoint and application routes to direct, private connectivity.
+
+|-r, --region
+|The AWS region (string) where your worker pool will be located. This argument overrides the AWS_REGION environment variable.
+
+|--service-cidr
+|Block of IP addresses (ipNet) for services. Example: 172.30.0.0/16
+
+
+|--subnet-ids
+|The subnet IDs (string) to use when installing the cluster. SubnetIDs must be in pairs with one private subnetID and one public subnetID per availability zone. Subnets are comma delimited. Example: ``--subnet-ids=subnet-1,subnet-2`. Leave the value empty for installer provisioned subnet IDs.
+
+|--version
+|The version (string) of OpenShift that will be used to install the cluster. Example: 4.3.10
+
+|--watch
+|Watches the cluster installation logs.
+|===
+
+.Optional arguments inherited from parent commands
+|===
+|Option |Definition
+
+|-h, --help
+|Shows help for this command.
+
+|debug
+|Enables debug mode.
+
+|-i, --interactive
+|Enables interactive mode.
+
+|--profile
+|Uses a specific AWS profile (string) from your credentials file.
+
+|-v, --v level
+|Log level for V logs.
+|===
+
+.Examples
+Create a cluster named 'mycluster'.
+
+[source,terminal]
+----
+$ rosa create cluster --cluster-name=mycluster
+----
+
+Create a cluster with a specific AWS region.
+
+[source,terminal]
+----
+$ rosa create cluster --cluster-name=mycluster --region=us-east-2
+----
+
+[id="rosa-create-idp_{context}"]
+== create idp
+
+Adds an identity provider (IDP) to define how users will log in to a cluster.
+
+.Syntax
+[source,terminal]
+----
+$ rosa create idp --cluster=<clusterName> [arguments]
+----
+
+.Arguments
+|===
+|Option |Definition
+
+|-c, --cluster
+|(Required) The name or ID (string) of the cluster to which the IDP will be added.
+
+|--ca
+|The path (string) to PEM-encoded certificate file to use when making requests to the server.
+
+|--client-id
+|The client ID (string) from the registered application.
+
+|--client-secret
+|The client secret (string) from the registered application.
+
+|--mapping-method
+|Specifies how new identities (string) are mapped to users when they log in. Default: claim
+
+|--name
+|The name (string) for the identity provider.
+
+|-t, --type
+|The type (string) of identity provider. Options: github, gitlab, google, ldap, openid
+|===
+
+.Github arguments
+|===
+|Option |Definition
+
+|--hostname
+|(Github) The optional domain (string) to use with a hosted instance of GitHub Enterprise.
+
+|--host-url
+|(Github) The host URL (string) of a GitLab provider. Default: https://gitlab.com
+
+|--organizations
+|(Github) Specifies the organizations for log in access. Only users that are members of at least one of the listed organizations (string) will be allowed to log in.
+
+|--teams
+|(Github) Specifies the teams for log in access. Only users that are members of at least one of the listed teams (string) will be allowed to log in. The format is <org>/<team>.
```

Comment:
* log in access -> login access
* will be -> are

Reply:
Done.